src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX = 100;
+    private static final int MIN = 1;
+    private static final int NUMBER = 4; // the limit of the number of figures
```

Comment:
```suggestion
    private static final int FIGURES_COUNT = 4;
```
краще уникати коментарів, коли зміст можна передати неймінгом

Reply:
В даному випадку неймінг константи - LIMIT_NUMBER_OF_FIGURE, був би коректним?